splitter.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -x
```

Comment:
In general, this script could use a header comment and some sugar sprinkled throughout to help guide people on why it's doing what it's doing.

For example: what is a `chunk`? How does that differ from a `split`? etc.